### PR TITLE
Oracle actor startup behavior

### PR DESCRIPTION
**Pull request by da-kami**

### CODE REVIEW

File: daemon/src/maker.rs [hunk 1]
```diff
@@ -265,6 +265,8 @@ async fn main() -> Result<()> {
                     cfds,
                 )));
 
+                oracle_actor_address.do_send(oracle::Sync).unwrap();
```
Comment (thomaseizinger) [inline daemon/src/maker.rs]: `do_send` will block the entire executor thread. Better use `send().await`.

**Review by thomaseizinger — CHANGES_REQUESTED** [review]
Thanks!

File: daemon/src/maker.rs (hunk 1 above)
Comment (da-kami) [reply daemon/src/maker.rs]: woops, thanks! will change :)

**Review by thomaseizinger — APPROVED** [review]

**Pull request merged**